.circleci/config.yml

```diff
@@ -85,18 +85,21 @@ commands:
     steps:
       - cmake_build
       - run: |
-          cd build && tools/json2json -h &&
+          cd build &&
+          tools/json2json -h &&
           ctest $CTEST_FLAGS -L acceptance &&
           ctest $CTEST_FLAGS -LE acceptance -E checkperf
 
   cmake_test_all:
```

Comment:
Hmmm... I think that "cmake_test_all" was meant to be "test everything including checkperf". Of course, we don't use it elsewhere... do we?

Reply:
Oh! I didn't realize we didn't use it. I guess we should either remove it or update it as is.

I just did a search for ctest to make this PR because I thought I saw a checkperf failure that wasn't obviously marked "checkperf." I could've just been wrong about it, though.